src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationOverlay.tsx

```diff
@@ -0,0 +1,128 @@
+// Libraries
+import React, {FC, useCallback, useContext, useState} from 'react'
+import {useDispatch} from 'react-redux'
+import {
+  Button,
+  ComponentColor,
+  ComponentStatus,
+  FlexBox,
+  FlexDirection,
+  Overlay,
+} from '@influxdata/clockface'
+
+// Components
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+import {InputWithLabel} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/InputWithLabel'
+import {ProviderChooser} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderChooser'
+
+// Styles
+import './CreateOrganizationOverlay.scss'
+import {
+  CreateOrgProvider,
+  CreateOrgContext,
+} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationContext'
+
+// Types
+import {RemoteDataState} from 'src/types'
+import {notify} from 'src/shared/actions/notifications'
+import {orgCreateSuccess} from 'src/shared/copy/notifications'
+
+const DEFAULT_ERROR_MESSAGE = 'This is a required field'
+const NAME_CONFLICT_MESSAGE =
+  'This organization name already exists. Please choose a different name.'
+
+const CreateOrgOverlay: FC = () => {
+  const {onClose} = useContext(OverlayContext)
+  const {orgName, changeOrgName, createOrg, createOrgLoadingStatus} =
+    useContext(CreateOrgContext)
+  const dispatch = useDispatch()
+  const [errorMessage, setErrorMessage] = useState(DEFAULT_ERROR_MESSAGE)
+  const [showError, setShowError] = useState(false)
+
+  const handleOrgNameChange = useCallback(
+    e => {
+      changeOrgName(e.target.value)
+      setShowError(false)
+      setErrorMessage(DEFAULT_ERROR_MESSAGE)
```

Comment:
Why are we setting `showError` to `false` here?

Reply:
as soon as we change org name, I think the error should go away.